crates/nu-engine/src/eval.rs

```diff
@@ -860,15 +864,17 @@ fn eval_element_with_input(
                 | Expr::StringInterpolation(_)
                 | Expr::Filepath(_),
             ) => {
+                // FIXME: currently save command only supports one `--append` argument
+                // it's required to support something like `--stderr-append` flag
```

Comment:
I don't understand what these FIXME comments mean. Can you explain it a little better?

Reply:
Oh, sorry, it's already solved..